ios/PurchasesHybridCommon/PurchasesHybridCommon/Purchases+HybridAdditions.swift

```diff
@@ -78,12 +79,44 @@ import RevenueCat
             }
         }
 
+        configurationBuilder = configurationBuilder.with(preferredLocale: preferredLocale)
```

Comment:
Hmm interesting... it seems here we're calling this even if `preferredLocale` is null, but we don't on Android. I think it's probably not needed so we can follow what we do in android?